packages/story-editor/src/components/canvas/karma/carousel.karma.js

```diff
@@ -74,9 +75,10 @@ describe('Carousel integration', () => {
   }
 
   async function clickOnThumbnail(index) {
+    await fixture.events.sleep(100);
     await fixture.editor.footer.carousel.waitReady();
     const thumb = fixture.editor.footer.carousel.pages[index];
-    thumb.node.scrollIntoView();
+    await thumb.node.scrollIntoView();
```

Comment:
Since when is [`scrollIntoView`](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) async?

Reply:
I also see `await vaPanel.panelTitle.scrollIntoView();` in `videoPoster.karma.js`, which would also be incorrect

Reply:
thanks, yah - removed both of 'em.